testing/shared/src/main/scala/org/typelevel/log4cats/testing/StructuredTestingLogger.scala

```diff
@@ -16,11 +16,12 @@
 
 package org.typelevel.log4cats.testing
 
+import cats.data.Chain
 import org.typelevel.log4cats.SelfAwareStructuredLogger
-import cats.effect.Sync
-import cats.implicits._
-import java.util.concurrent.atomic.AtomicReference
+import cats.effect.{Ref, Sync}
+import cats.implicits.*
```

Comment:
?
```suggestion
import cats.syntax.all._
```

Reply:
Not sure how that sneaked in there, thanks for the catch.